src/stories/10_Introduction/_AMP_Story_Hello_World.html

```diff
@@ -0,0 +1,109 @@
+<!-- 
+  ## Introduction
+
+  Learn how to create your first AMP Story. The [amp-story](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md) extension provides a new format for displaying visual content. 
+-->
+<!-- -->
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- AMP Stories are written using AMPHTML and they use their own AMP extension: `amp-story`. The first step is to import the `amp-story` in the header. -->
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+    <!-- AMP Stories can make use of other AMP extensions such as `amp-video`. Howeever, they support only a subset of the available AMP extensions. You can find a full list of the supported extensions [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md#children-of-amp-story-grid-layer). -->
+    <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+    <title>Stories in AMP - Hello World</title>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link rel="canonical" href="/stories/introduction/amp_story_hello_world/">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <!-- Stories can be styled using CSS -->
+    <style amp-custom>
+      body {
+        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI ", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji ", "Segoe UI Emoji ", "Segoe UI Symbol ";
+      }
+      amp-story-page * {
+        color: white;
+        text-align: center;
+      }
+      [template=thirds] {
+        padding: 0;
+      }
+    </style>
+  </head>
+
+  <body>
+    <!-- ## Writing the Story -->
+    <!-- Stories are declared inside the `amp-story` tag. You can link to other Stories on the last page of your story by linking to a [bookend JSON config](/json/bookend.json). -->
+    <amp-story standalone bookend-config-src="<%host%>/json/bookend.json">
```

Comment:
It's too bad you don't show the last screen that's the bookend.
Also, perhaps change "last page" to "last screen" because they don't actually create a page, it's the "wrap-up" screen.

Reply:
fixed screen, will show a bookend preview once I found the time to implement jumping to the bookend.